tensorflow/python/ops/image_ops_impl.py

```diff
@@ -1867,13 +1867,21 @@ def rgb_to_grayscale(images, name=None):
   last dimension of the output is 1, containing the Grayscale value of the
   pixels.
 
+  Usage Example:
+    ```
+    >>> import tensorflow as tf
+    >>> x = tf.random.normal(shape=(256, 256, 3))
+    >>> tf.image.rgb_to_grayscale(x)
+    ```
```

Comment:
Please include the result of the operation so this becomes a doctest.

To make the doctest meaningful replace the large image with random values with a small (1x1?) image with known values.